components/docs-chef-io/content/automate/chef_automate_profile.md

```diff
@@ -0,0 +1,55 @@
++++
+title = "User Profile"
+
+draft = false
+
+gh_repo = "automate"
+[menu]
+  [menu.automate]
+    title = "User Profile"
+    parent = "automate/settings"
+    identifier = "automate/settings/chef_automate_profile.md User Profile"
+    weight = 120
++++
+
+## Overview
+
+Chef Automate users can manage their own display name, date/time format, and password. Select the user icon in the top navigation bar, and then select **Profile** from the drop-down menu.
+
+{{< figure src="/images/automate/chef-automate-profile.png" alt="Profile Section in Chef Automate">}}
+
+Select **Profile** to view the details with respect to your _username_. The **Details** tab lets you update your _Display Name_ and the _Time Format_, whereas the **Reset Password** tab lets you update your password.
```

Comment:
```suggestion
Select **Profile** to view your _username_. The **Details** tab lets you update your _Display Name_ and the _Time Format_, whereas the **Reset Password** tab lets you update your password.
```
```suggestion
Select **Profile** to view the details with respect to your _username_. The **Details** tab lets you update your _Display Name_ and the _Time Format_, whereas the **Reset Password** tab lets you update your password.
```

Reply:
Could you disambiguate between the `username` and the `Display Name`.  I don't know the difference between the two from this doc.

Reply:
Yes!
`username` is the login name which we need to login into our account, whereas display name is the name displayed in profiles.